### PR TITLE
Updates to half and third width link blob documentation

### PR DESCRIPTION
**Pull request by nataliafitzgerald**
The addition of documentation for the half width and third width link blobs is needed so that Wagtail users are aware of content strategy and design considerations.

## Additions
* Content updates to half width link blob page
* Creation of third width link blob page

## Review
@vczosek 

[Preview this PR without the whitespace changes](?w=0)

## Screenshots
screencapture-nataliafitzgerald-github-io-design-manual-page-components-half-width-link-blob-html-1481137978773
screencapture-nataliafitzgerald-github-io-design-manual-page-components-third-width-link-blob-html-1481137937725

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: page-components/half-width-link-blob.md [hunk 1]
```diff
@@ -28,12 +22,13 @@ A pair of half-width link blobs, as seen on a landing page template:
...
-- Default option for providing succinct descriptions and navigation path to child pages from a parent page.
+#### When to use 
+- Default option for providing succinct descriptions and navigation paths to child pages from a parent page.
 - Ideally, when an even number of these components can be featured together.
 
-##### When other options are better 
-- When content requires imagery. Suggest using the 50/50 image and text or 25/27 image and text component instead.
+#### When other options are better 
+- When the number of components would be better featured in multiples of 3. Suggest using third-width link blob.
+- When content requires imagery. Suggest using the [50/50 image and text]({{site.baseurl}}//page-components/50-50.html) or [25/75 image and text]({{site.baseurl}}//page-components/25-75.html) components instead.
```
Comment (virginiacc) [inline page-components/half-width-link-blob.md]: It looks like there's an extra slash after `{{site.baseurl}}` in these links (and the ones on the third-width page.)

File: page-components/third-width-link-blob.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+- [Content guidelines](#content-guidelines)
+- [Style](#style)
+{: class="toc"}
+
+<div>
+The third-width link blob provides users with supplemental content or suggested navigation paths to sibling pages. The three-column layout utilizes extra screen real estate found at higher breakpoints.
+{: class="lead-in"}
+
+<nomarkdown>
+<img src="https://raw.githubusercontent.com/nataliafitzgerald/design-manual/gh-pages/static/img/third_width_link_blob/third_width_link_blob_intro.png" alt="Mockup" height="100%" width="100%">
```
Comment (virginiacc) [inline page-components/third-width-link-blob.md]: We can use internal paths for the added images:
```
<img src="{{site.baseurl}}/static/img/third_width_link_blob/third_width_link_blob_behavior_1.png" ...>
```

**Review by virginiacc — CHANGES_REQUESTED** [review]
@nataliafitzgerald This looks good to me! I just had a few small suggestions for the links/images...

File: page-components/half-width-link-blob.md (hunk 1 above)
Comment (jimmynotjim) [reply page-components/half-width-link-blob.md]: This seems to be happening throughout this pull request.

**Comment by nataliafitzgerald** [comment]
@virginiacc @jimmynotjim 
I made some updates to my file to reflect the comments above. Let me know if there is anything else I need to do to share this out. Still learning the process. Thanks!

**Comment by jimmynotjim** [comment]
@nataliafitzgerald awesome, this is working locally so I'll fix the conflicts and get it merged. Thanks for getting it updated.

**Comment by nataliafitzgerald** [comment]
@willbarton
This documentation is finally wrapped up and live on the CFPB Design Manual.

**Pull request merged**